run_dir/static/js/fc_trend_plot.js

```diff
@@ -201,10 +203,12 @@ function build_series(data, key, name, display_by, filter_inst_type, filter_inst
           if (data[d].instrument.indexOf('M') != -1){
               series_name = "MiSeq";
           }else if (data[d].instrument.indexOf('A') != -1){
-              series_name = "NovaSeq";
-          }else if (data[d].instrument.indexOf('N') != -1){
-              series_name = "NextSeq";
-            }else{
+              series_name = "NovaSeq 6000";
+          }else if (data[d].instrument.indexOf('VH') != -1){
+              series_name = "NextSeq 2000";
+          }else if (data[d].instrument.indexOf('LH') != -1){
+              series_name = "NovaSeq XPlus";
```

Comment:
NovaSeq X Plus or NovaSeq XPlus ?

Reply:
NovaSeqXPlus is how we've decided it should be named.

Reply:
Guess we'll go with "NovaSeq X Plus" :-)